lib/coherence/router.ex

```diff
@@ -86,62 +88,62 @@ defmodule Coherence.Router do
       if mode == :all or mode == :public do
         Enum.each([:new, :create], fn(action) ->
           if Coherence.Config.has_action?(:authenticatable, action) do
-            resources "/sessions", Coherence.SessionController, only: [action]
+            resources "#{routes.sessions}", Coherence.SessionController, only: [action]
```

Comment:
Why string interpolation here? These should already be strings so I don't see the values occurring the overhead of string interpolation. If there is really a need, then perhaps `to_string/1` might be a better solution.